doc/plugin_server_nodeattestor_x509pop.md

```diff
@@ -47,26 +47,20 @@ A sample configuration:
 
 ## SVID Path Prefix
 
-When mode="spiffe", the SPIFFE ID being exchanged must be prefixed by the specified svid_prefix. The prefix will be removed from the .SVIDPathTrimmed property before sending to the
-agent path template. If set to "", all prefixes are allowed and you will want to do limiting logic in in the agent_path_template.
-
-Example: if your trust domain is example.com and svid_prefix = the default of /spire-exchange, and agent path template is the default,
+When `mode="spiffe"` the SPIFFE ID being exchanged must be prefixed by the specified `svid_prefix`. The prefix will be removed from the `.SVIDPathTrimmed` property before sending to the agent path template. If `svid_prefix` is set to `""`, all prefices will allowed, and the limiting logic will have to be implemented in the `agent_path_template`.
 
-spiffe://example.com/spire-exchange/testhost will render out to spiffe://example.com/spire/agent/x509pop/testhost
-
-If spiffe://example.com/other/testhost is given, it wont match the svid_prefix and it will be rejected.
+**Example:** If your trust domain is example.com and `svid_prefix` is set to its default value `/spire-exchange`, and [agent_path_template](#agent-path-template) is the default too, then the SPIFFE ID from the x509 identity `spiffe://example.com/spire-exchange/testhost` will be exchanged for `spiffe://example.com/spire/agent/x509pop/testhost`. If a SPIFFE ID with a different prefix is given, for example `spiffe://example.com/other/testhost`, it will not match the `svid_prefix` and will be rejected.
 
 ## Agent Path Template
 
-The agent path template is a way of customizing the format of generated SPIFFE IDs for agents.
-
-If using ca_bundle_path(s), the default is:
-"{{ .PluginName }}/{{ .Fingerprint }}"
+Specifying the value of `agent_path_template` provides a way of customizing the format of generated SPIFFE IDs for agents. The default format for every mode is shown below
 
-If using spire_trust_bundle, the default exchanges an SVID under `/spire-exchange/*` for `/spire/agent/x509pop/*`, via:
-"{{ .PluginName }}/{{ .SVIDPathTrimmed }}"
+|`mode`|`agent_path_template`|
+|------|---------------------|
+| `spiffe` | `{{ .PluginName }}/{{ .SVIDPathTrimmed }}`|
+| `external_pki` |  `{{ .PluginName }}/{{ .Fingerprint }}` |
```

Comment:
```suggestion
| `mode`         | `agent_path_template`                      |
|----------------|--------------------------------------------|
| `spiffe`       | `{{ .PluginName }}/{{ .SVIDPathTrimmed }}` |
| `external_pki` | `{{ .PluginName }}/{{ .Fingerprint }}`     |
```
nit, can we keep the table formatting aligned for maintainability?

Reply:
Thank you, addressed both suggestions!